client/blocks/comments/post-comment.jsx

```diff
@@ -168,10 +168,20 @@ class PostComment extends Component {
 						: <Gravatar user={ comment.author } /> }
 
 					{ authorUrl
-						? <a href={ authorUrl } className="comments__comment-username" onClick={ this.handleAuthorClick }>
+						? <a
+							href={ authorUrl }
+							className="comments__comment-username"
+							onClick={ this.handleAuthorClick }
+							name={ `comment-${ this.props.commentId }` }
```

Comment:
looks equivalent for this case, but i'll switch it over.  that was an interesting read, I had no idea how the spec worked before.